.github/workflows/main.yml

```
@@ -13,14 +13,19 @@ jobs:
         include:
           - php: 7.4
             symfony: "^4.4"
+            dependency-versions: "lowest"
           - php: 8.1
             symfony: "^5.4"
+            dependency-versions: "higest"
```

Typo (multiple times):
```suggestion
            dependency-versions: "highest"
```